apps/ledger-live-mobile/src/components/BuyDeviceBanner.tsx

```diff
@@ -147,7 +161,7 @@ export default function BuyDeviceBanner({
             type="color"
             Icon={Icons.ArrowRightMedium}
             iconPosition="right"
-            onPress={handleSetupCtaOnPress}
+            onPress={() => pressMessage()}
```

Comment:
you can just pass `pressMessage` without recreating the function reference

Reply:
We missed it ...